packages/grafana-ui/src/components/Logs/LogRows.tsx

```diff
@@ -24,9 +24,6 @@ export interface Props extends Themeable {
   logsSortOrder?: LogsSortOrder | null;
   allowDetails?: boolean;
   previewLimit?: number;
-  // Passed to fix problems with inactive scrolling in Logs Panel
-  // Can be removed when we unify scrolling for Panel and Explore
-  disableCustomHorizontalScroll?: boolean;
```

Comment:
We can finally get rid of this and we are using Custom Scrollbar in Logs component